Gemfile.lock

```diff
@@ -81,7 +81,7 @@ GEM
     ethon (0.11.0)
       ffi (>= 1.3.0)
     execjs (2.7.0)
-    factory_girl (4.9.0)
+    factory_bot (4.8.2)
```

Comment:
factory_bot 4.9.0 does not seem to be available for me.. not sure why?

Reply:
oh, the blog post explains, 4.9.0 is factory_girl only, and adds the deprecation warning that this PR removes.